app/views/alchemy/essences/shared/_essence_picture_tools.html.erb

```diff
@@ -33,7 +33,7 @@
 ) %>
 
 <%= link_to(render_icon(:link), '', {
-  onclick: 'd = new Alchemy.LinkDialog(this); d.open(); return false;',
+  onclick: 'window.dialog = new Alchemy.LinkDialog(this); window.dialog.open(); return false;',
```

Comment:
You don't need to attach the dialog into the global DOM, because Alchemy already adds the dialog into `Alchemy.currentDialogs` Array and you can get the current/latest one with `Alchemy.currentDialog()`